macros/parsers/parserCheckboxList.pl

```diff
@@ -345,18 +364,27 @@ sub getCorrectChoices {
 		}
 
 		for (0 .. $#{ $self->{orderedChoices} }) {
-			if ($value eq $self->{orderedChoices}[$_] || $value eq ($self->{labels}[$_] || '')) {
+			if ($value eq $self->{orderedChoices}[$_]
+				|| $value eq ($self->{labels}[$_] || ''))
+			{
 				push(@{ $self->{data} }, $self->{values}[$_]);
 				last;
 			}
 		}
 	}
 
 	# Sort the correct choices into display order.
-	$self->{data} =
-		[ main::PGsort(sub { $self->getIndexByValue($_[0]) < $self->getIndexByValue($_[1]) }, @{ $self->{data} }) ];
-
-	Value::Error('The correct choices must be among the label values') unless @{ $self->{data} };
+	$self->{data} = [
+		main::PGsort(
+			sub {
+				$self->getIndexByValue($_[0]) < $self->getIndexByValue($_[1]);
+			},
+			@{ $self->{data} }
+		)
+	];
+
+	Value::Error('The correct choices must be among the label values')
+		unless @{ $self->{data} };
```

Comment:
I personally prefer keeping this on line.  For the `statement if|unless conditional` construct, I generally prefer that to be on one line if it fits well within the 120 character limit.